### PR TITLE
Add the survey_option view column type.

### PR DESCRIPTION
**Pull request by niklasva82**
Adds a survey option view column type.

If a person has checked the specified option in any of their submissions, it will indicate true, otherwise false.

Clicking a cell shows either the latest submission with a true submission or the latest submission if not true, if the person has submitted the survey.

### CODE REVIEW

**Comment by niklasva82** [comment]
Screenshot from 2022-11-24 13-26-59

File: src/components/misc/personViews/cells/SurveyOptionCell.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import cx from 'classnames';
+import React from 'react';
+
+export default function SurveyOptionCell(props) {
+    const checked = props.content.some((submission) => submission.selected);
+
+    // Find the latest submission where selected is true, or failing that, the latest submission
```
Comment (richardolsson) [inline src/components/misc/personViews/cells/SurveyOptionCell.jsx:7]: Great comment! Always love a comment to accompany a reduce. 😊

**Review by richardolsson — COMMENTED** [review]
This looks all good to me. Waiting until the feature has been merged in the API before merging this though.

**Comment by richardolsson** [comment]
@niklasva82 There are merge conflicts here. Could you resolve them? 🙏

**Comment by niklasva82** [comment]
> @niklasva82 There are merge conflicts here. Could you resolve them? pray

The conflicts have been resolved!

**Comment by richardolsson** [comment]
PR approved, but still waiting for finalized API implementation in Zetkin Platform before merging.

**Pull request merged**